src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ClippedBases.java

```diff
@@ -0,0 +1,41 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
+
+import com.google.common.primitives.Doubles;
+import com.google.common.primitives.Ints;
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.utils.GATKProtectedMathUtils;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.read.AlignmentUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.apache.commons.math3.stat.descriptive.rank.Median;
+
+
+import java.util.List;
+import java.util.OptionalDouble;
+import java.util.OptionalInt;
+
+/**
+ * Created by David Benjamin on 3/20/17.
+ */
+public class ClippedBases extends PerAlleleAnnotation {
+    private static byte BASE_QUAL_THRESHOLD = 25;
+
+    public static final String KEY = "MCL";
```

Comment:
Yes, but that class is in gatk public.  Later I will move all of this to `GATKVCFConstants`.